engine-agent/pom.xml

```diff
@@ -34,6 +34,14 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
 
+    <!--  Usually we don't want to provide an slf4j implementation in this project. This modules
+      is the only exception to this rule. This is because we need to provide the engine with an
+      implementation, and since this runs in a docker container the user won't be providing it. -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
```

Comment:
:wrench: I guess this could be runtime scope. And then you could configure dependency plugin to ignore runtime dependencies